corehq/apps/accounting/enterprise.py

```diff
@@ -137,9 +137,19 @@ def __init__(self, account_id, couch_user):
     def headers(self):
         headers = super(EnterpriseWebUserReport, self).headers
         return [_('Name'), _('Email Address'), _('Role'), _('Last Login [UTC]'),
-                _('Last Access Date [UTC]')] + headers
+                _('Last Access Date [UTC]'), _('Status')] + headers
 
     def rows_for_domain(self, domain_obj):
+
+        def _get_role_name(role):
+            if role:
+                if role == 'admin':
+                    return role
+                else:
+                    role_id = role[len('user-role:'):]
```

Comment:
Nitpick: this would be cleaner as a function instead of having the magic string here. `unqualify_id`?